### PR TITLE
Set citra-qt project as default StartUp Project in Visual Studio

### PR DESCRIPTION
**Pull request by FreddyFunk**
Set citra-qt project as default StartUp Project in Visual Studio

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/4214)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -383,6 +383,9 @@ add_subdirectory(externals)
 add_subdirectory(src)
 add_subdirectory(dist/installer)
 
+# Set citra-qt project as default StartUp Project in Visual Studio
```
Comment (adityaruplaha) [inline CMakeLists.txt]: Ultra minor nitpick: 
'U' shouldn't be in caps.

File: CMakeLists.txt [hunk 2]
```diff
@@ -383,6 +383,9 @@ add_subdirectory(externals)
...
+set_property(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR} PROPERTY VS_STARTUP_PROJECT citra-qt)
```
Comment (zhaowenlan1779) [inline CMakeLists.txt]: Should we only set this when qt is enabled? (`ENABLE_QT`)

File: CMakeLists.txt (hunk 1 above)
Comment (adityaruplaha) [reply CMakeLists.txt]: Nvm then.

**Comment by FreddyFunk** [comment]
Adressed feedback from @zhaowenlan1779

**Comment by FreddyFunk** [comment]
mingw build on AppVeyor canceled after 29 seconds with error code "ServiceUnavailable". May need a restart.

**Pull request merged**